index.html

```diff
@@ -67,31 +67,31 @@ <h2>Interests</h2>
             <section id="contact">
                 <h2 class="contact-item">Contact</h2>
                 <address class="contact-item">
-                    <div class="address-item btn">
+                    <div class="address-item">
                         <i class="fas fa-phone"></i>
                         <p>+49 (0)170 233XXXX</p>
                     </div>
-                    <div class="address-item btn">
+                    <div class="address-item">
                         <i class="fas fa-envelope"></i>
-                        <p>mberger83[at]yahoo.de</p>
+                        <p><a href="mailto:mberger83@yahoo.de">mberger83[at]yahoo.de</a></p>
                     </div>
                 </address>
-            </section>
+            </section> 
             <section id="works">
                 <h2>Recent Works</h2>
                 <div class="works-ctn">
-                    <div class="works-item">
-                        <img src="github-logo.svg" alt="" width="200" height="200">
-                        <a href="">github</a>
-                    </div>
-                    <div class="works-item">
-                        <img src="codepen-logo.svg" alt="" width="200" height="200">
-                        <a href="">codepen</a>
-                    </div>
-                    <div class="works-item">
-                        <img src="linkedin-logo.svg" alt="" width="200" height="200">
-                        <a href="">linkedin</a>
-                    </div>
+                        <a class="works-item" href="https://github.com/MichiBerger">
+                            <img src="github-logo.svg" alt="" width="200" height="200">
+                            <h3>github</h3>
+                        </a>
+                        <a class="works-item" href="">
+                            <img src="codepen-logo.svg" alt="" width="200" height="200">
+                            <h3>codepen</h3>
+                        </a>
+                        <a class="works-item" href="https://www.linkedin.com/in/michael-berger-697321158/">
+                            <img src="linkedin-logo.svg" alt="" width="200" height="200">
+                            <h3>linkedin</h3>
+                        </a>
```

Comment:
here you should check the indentation if this was not on purpose ;)

Reply:
perfect thx mirco :smile it was not on purpose 👍 i change it straigth away! thx